lib/text/unicodecharacters.flow

```diff
@@ -185,10 +186,17 @@ ucLookupTable(table, c, fallback) {
 			subtable[lo]
 		else fallback;
 	} else fallback;
-} 
+}
 
+UnicodeDirectionCache = {
```

Comment:
This implies a runtime initialization cost for all flow programs that include this. Is that intended? Maybe use onlyOnce?